mac/io.parity.parity-ethereum.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>io.parity.parity</string>
+    <key>ProgramArguments</key>
+    <array>
+        <string>/usr/local/libexec/parity</string>
+        <string>--mode</string>
+        <string>passive</string>
+        <string>--warp</string>
+    </array>
+    <key>KeepAlive</key>
+    <true/>
+    <key>StandardErrorPath</key>
+    <string>/var/log/parity.log</string>
+</dict>
```

Comment:
should we also add `--warp` and `--sync` to homebrew plist?
https://github.com/ethcore/homebrew-ethcore/blob/master/parity.rb#L33